integration-tests/tests/src/tests/credentials/jwt-encode/index.d.ts

```diff
@@ -0,0 +1,31 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+declare module "jwt-encode" {
```

Comment:
I can't remember why this was originally needed in realm-web, but it there does seem to be a `@types/jwt-encode` package available for this: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jwt-encode/index.d.ts